pandas/_libs/tslibs/conversion.pyx

```diff
@@ -104,12 +105,6 @@ cdef int64_t cast_from_unit(
         int64_t m
         int p
 
-    m, p = precision_from_unit(unit, out_reso)
-
-    # just give me the unit back
-    if ts is None:
-        return m
```

Comment:
is it because this was uncovered anyway?

Reply:
correct